### PR TITLE
Add ability to customize colors for disabled toggle

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
No ability to customize colors for disabled Toggle

## What is the new behavior?
Added ability to customize colors for disabled Toggle

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by amcdnl — APPROVED** [review]

File: src/form/Toggle/Toggle.module.css [hunk 1]
```diff
@@ -15,7 +10,20 @@
 
   &.disabled {
     cursor: not-allowed;
-    opacity: 0.8;
+    opacity: var(--toggle-disabled-opacity);
```
Comment (amcdnl) [inline src/form/Toggle/Toggle.module.css:13]: Let's add a default here.

```suggestion
    opacity: var(--toggle-disabled-opacity, 0.8);
```

**Review by amcdnl — CHANGES_REQUESTED** [review]
Minor tweaks required

**Comment by amcdnl** [comment]
Need to update docs too.

**Review by amcdnl — COMMENTED** [review]
also need to update the Toggle.mdx

**Comment by evgenoid** [comment]
> also need to update the Toggle.mdx

Done

**Review by amcdnl — APPROVED** [review]

**Pull request merged**